src/renderer.tsx

```diff
@@ -63,6 +64,8 @@ try {
   console.error(`PastelDB.getDatabaseInstance error: ${error.message}`)
 }
 
+log.transports.console.level = false
```

Comment:
Let's have console level = false only in production mode, to be able to see the log during development

And file.level = false for not in production, as we can see log in console during development and no need for log files

Reply:
Fixed